cms/djangoapps/contentstore/xblock_storage_handlers/view_handlers.py

```diff
@@ -87,6 +88,11 @@
 
 CREATE_IF_NOT_FOUND = ["course_info"]
 
+# List of categories to check for presence in the children of the XBlock.
+# This list is used to determine if all of the specified categories are absent
+# in the categories of the children XBlock instances otherwise icon class variable will be set to `None`.
+CATEGORIES_WITH_ABSENT_ICON = ["split_test"]
```

Comment:
This seems difficult to maintain, but I can't think of a cheaper workaround.  The question is if there's a more architecturally sound way of doing it, and if we want to hold off on merging this until we find one.

@kdmccormick or @feanil, thoughts?

Reply:
@arbrandes Agreed, we should not hard code block types if at all possible. There are places that do it already, but I would rather avoif adding more.

The comment is somewhat confusing to me. What is the meaning of an item being in this list? Is it that is type should be ignored when determining the icon?